template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/StarterWeb-CSharp/Areas/IdentityService/ClientInfoProvider.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Identity.Service;
+using Microsoft.Extensions.Options;
+using Microsoft.IdentityModel.Protocols.OpenIdConnect;
+using Microsoft.IdentityModel.Tokens;
+using Newtonsoft.Json;
+
+namespace Company.WebApplication1.Identity
+{
+    public class ClientInfoProvider : IAuthorizationResponseParameterProvider, ITokenResponseParameterProvider
```

Comment:
User wouldn't change it. Moving this into extensions https://github.com/aspnet/IdentityService/issues/84